src/renderer/services/Eth1ICGKeystore.ts

```diff
@@ -0,0 +1,101 @@
+import { ICGKeystore } from './interface';
+import { Keypair } from '@chainsafe/bls/lib/keypair';
+import * as fs from 'fs';
```

Comment:
import fs from 'fs' 
should work

Reply:
Module '"fs"' has no default export. Issue https://github.com/Microsoft/TypeScript/issues/3337

Reply:
Getting error `Module '"fs"' has no default export.`

Reply:
interesting. I would swear  it works on another project. Well use `import {writeFileSync} from "fs"` then :)